nautobot/core/utils/module_loading.py

```diff
@@ -24,6 +24,15 @@ def _temporarily_add_to_sys_path(path):
         sys.path = old_sys_path
 
 
+def clear_module_from_sys_modules(module_name):
+    """
+    Recursively remove the module and all its submodules from sys.modules.
```

Comment:
```suggestion
    Remove the module and all its submodules from sys.modules.
```

No recursion involved :)